src/libraries/libraries-packages.proj

```diff
@@ -18,6 +18,7 @@
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" Condition="'$(BuildAllOOBPackages)' == 'true'" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
     <!-- If setting BuildAllOOBPackages to false, add bellow the individual OOB packages you want to continue to build -->
+        <ProjectReference Include="$(PkgDir)runtime.native.System.IO.Ports\runtime.native.System.IO.Ports.proj" />
```

Comment:
nit: indentation

Reply:
Shouldn't we also build System.IO.Ports so that it gest a reference to the new runtime.System.IO.Ports?